### ISSUE TITLE
`form.submittedSuccessfully` not holding the correct value

### ISSUE THREAD

**Issue by lindseydiloreto** [solspace/craft-express-forms#9]
It's possible that I'm doing something wrong, but this doesn't seem to be working...

```twig
{{ form.openTag({ return: 'get-a-quote/thanks' }) }}
```

It doesn't redirect. The form successfully submits, then it just comes back to the same page.

Similarly, the `form.submittedSuccessfully` value is not being marked as `true`, despite the fact that the data has been recorded, and an email has been sent out.

**Comment by lindseydiloreto** [comment]
Ok, this diagnosis might be a little off, but something squirrelly is _definitely_ going on.

I switched it up so that the page simply redirects back on itself. There's no need for a separate page, because (in theory) `form.submittedSuccessfully` will recognize that the form was submitted and handle it appropriately.

```twig
{{ form.openTag() }}
```

Assuming `form.submittedSuccessfully` registers as `true`, I'll just show a "thank you" message instead of the form.

**Steps to reproduce:**

1. When I complete & submit the form **the first time**...

2. It returns to the form with all of the data completely filled out. The data got stored, the email got sent, but the form reappeared fully populated... `form.submittedSuccessfully` is erroneously `false` at this point.

3. So I refreshed the browser (at which point the browser confirms that I want to resend the POST data)...

4.  _Then_ it proceeds to the "thank you" version of the page. The data got stored _again_, and the email got sent _again_.

5. Finally, I click the nav link to the form page that I'm already on, and it _again_ shows the "thank you" version of the page (instead of a fresh form). `form.submittedSuccessfully` is erroneously `true` at this point.

**To recap:**
 - Form needs to be submitted twice.
 - `form.submittedSuccessfully` registers as `true` twice.

I know this sounds a little crazy. Feel free to follow up with me with any questions.

**Comment by kjmartens** [comment]
Shoot, I'm really sorry about this. I think the latest update we made broke the submitting process. I can see in my testing that return URL no longer appears to work, the first submit does not clear the form, and the flash success doesn't seem reliable.

We'll have fixes out for all of these issues tomorrow morning. Thanks for reporting, and sorry again for the trouble. 😟

**Comment by lindseydiloreto** [comment]
No problem, thanks for being so quick and attentive about it!

It's the last piece of the puzzle on this Craft 3 update I'm doing, but I can kick the can and launch it all tomorrow. 🙂

**Comment by kjmartens** [comment]
Thanks!

**Comment by kjmartens** [comment]
This should now be fixed in v1.0.0-beta.4 :)

**Comment by lindseydiloreto** [comment]
Looks like this fix is working great! Thanks @kjmartens! 🍺

**Comment by kjmartens** [comment]
Thanks @lindseydiloreto!

**Comment by lindseydiloreto** [comment]
(And thanks also @gustavs-gutmanis! 🍺)

**Comment by danvolkens** [comment]
Apologies for opening this, but I seem to have the same problem on the latest install. I'm setting a url for the form to redirect to upon submission (thank you page) but it appears to only return to the form url and use the flash status message.
